mailme.go

```diff
@@ -111,20 +117,7 @@ func (t *TemplateCache) Set(key, value string, expirationTime time.Duration) (*t
 }
 
 func (t *TemplateCache) fetchTemplate(url string, triesLeft int) (string, error) {
-	client := http.Client{
-		Transport: &http.Transport{
-			DialContext: nfhttp.SafeDialContext((&net.Dialer{
-				Timeout:   30 * time.Second,
-				KeepAlive: 30 * time.Second,
-				DualStack: true,
-			}).DialContext),
-			MaxIdleConns:          100,
-			IdleConnTimeout:       90 * time.Second,
-			TLSHandshakeTimeout:   10 * time.Second,
-			ExpectContinueTimeout: 1 * time.Second,
-		},
-	}
-
+	client := nfhttp.SafeHTTPClient(http.DefaultClient, t.logger)
```

Comment:
The problem with the default client is that it doesn't have any concepts of timeouts. I don't think it is a huge issue but something to be aware of.

Reply:
See above comments :point_up: